spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -24,43 +24,43 @@
 
   it "accepts options for the validator" do
     create_schema("foo_schema", {
-      "type" => "object",
-      "required" => [
+      "type": "object",
+      "required": [
         "id",
       ],
-      "properties" => {
-        "id" => { "type" => "number" },
-        "title" => {"type" => "string"},
+      "properties": {
+        "id": { "type": "number" },
+        "title": {"type": "string"},
       },
-      "additionalProperties" => false,
+      "additionalProperties": false,
     })
 
-    expect(response_for({ "id" => 1, "title" => "bar" })).
+    expect(response_for({ "id": 1, "title": "bar" })).
       to match_response_schema("foo_schema", strict: false)
   end
 
   it "fails when the body contains a property with the wrong type" do
     create_schema("foo_schema", {
-      "type" => "object",
-      "properties" => {
-        "foo" => { "type" => "string" },
+      "type": "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.